doc/api/buffer.md

```diff
@@ -508,7 +508,7 @@ added: v5.10.0
 -->
 
 * `size` {Integer} The desired length of the new `Buffer`
-* `fill` {string | Buffer | Integer} A value to pre-fill the new `Buffer` with.
+* `fill` {string|Buffer|Integer} A value to pre-fill the new `Buffer` with.
```

Comment:
@silverwind in your [commit](https://github.com/nodejs/node/commit/9be03a2903a0196212868d9e4282d638355828e7#diff-bfa39b8453bf073f2b2cbbf0f2dd8040R7) that changed the case in the doctool, `Integer` was changed to `integer`. Either way it doesn't seem right to me that number and integer have different cases.